app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 15
+    number_of_parts = 3
+    result = split_integer(value, number_of_parts)
+    assert sum(result) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 20
+    number_of_parts = 5
+    result = split_integer(value, number_of_parts)
+    assert len(result) == number_of_parts
+    assert min(result) == max(result)
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    value = 8
+    number_of_parts = 1
+    result = split_integer(value, number_of_parts)
+    assert result == [value]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    value = 25
+    number_of_parts = 4
+    result = split_integer(value, number_of_parts)
```

Comment:
Do not create additional variables, pass values directly to the function and make required actions in the `assert`.